website/registries/utils.py

```diff
@@ -10,7 +10,7 @@ def get_campaign_schema(campaign):
         raise ValueError('campaign must be one of: {}'.format(', '.join(REG_CAMPAIGNS.keys())))
     schema_name = REG_CAMPAIGNS[campaign]
 
-    return RegistrationSchema.objects.get(name=schema_name, schema_version=2)
+    return RegistrationSchema.objects.get(name=schema_name, schema_version=3)
```

Comment:
I think I'm not understanding this properly. OSF Preregistration is on version 3 with this change, but Registered Reports is on 4. How does this get the proper version?

Reply:
Added a `REG_CAMPAIGNS_VERSION` dict to get the correct version number.